assets/styles/components/wordpress/_gallery.scss

```diff
@@ -1,11 +1,26 @@
 .gallery {
+  $margin: 20px;
+  $wrapper-padding: map-get($wrapper, 'padding');
+  $padding-small: map-get($wrapper-padding, 'small') * 2;
+  $padding-medium: map-get($wrapper-padding, 'medium') * 2;
+  $width: 152px;
```

Comment:
The value is now calculated: https://github.com/trendwerk/trendpress/pull/750/commits/a66dc2d2d26781601cfdfc17b5c23c228f9531c7